12-offer-encoding.md

```diff
@@ -605,30 +655,21 @@ A writer of an invoice:
     - MUST include `invoice_blindedpay` with exactly one `blinded_payinfo` for each `blinded_path` in `paths`, in order.
     - MUST set `features` in each `blinded_payinfo` to match `encrypted_data_tlv`.`allowed_features` (or empty, if no `allowed_features`).
     - SHOULD ignore any payment which does not use one of the paths.
-  - if responding to an `invoice_request`:
-    - if `invoice_request_payer_key` and offer are identical to a previous `invoice_request`:
-      - MAY simply reuse the previous invoice.
-    - otherwise:
-      - MUST NOT reuse a previous invoice.
-    - if `invoice_request_amount` is present:
-      - MUST set `invoice_amount` to `invoice_request_amount`
-    - otherwise: (no `invoice_request_amount`, so must have `offer_amount`):
-      - MUST set `invoice_amount`.`msat` to the *base invoice amount*.
```

Comment:
Why drop this?

Reply:
Previously, the base invoice amount was calculated in the invoice_request parsing, and *used* in invoice writing.  Which was just weird.

Now we reflect all the fields, the invoice writing has all the information to calculate this where it's needed.  So it was moved.